src/tactic/ui/startup/security_wdg.py

```diff
@@ -422,8 +443,13 @@ def execute(my):
             new_group = Common.get_filesystem_name(new_group)
 
             group = SearchType.create("sthpw/login_group")
-            group.set_value("login_group", new_group)
-            group.set_value("project_code", project_code)
+            
+            if my.kwargs.get("checked") in [True, "true", "True"]:
+                group.set_value("project_code", project_code)
+                group.set_value("login_group", new_group)
+            else:
+                group.set_value("login_group", title)
```

Comment:
if they enter vfx/admin, it should take it as is.. but now title = admin only. That's not right